ui/views_test.py

```diff
@@ -439,52 +441,6 @@ def test_collection_viewset_detail_as_superuser(logged_in_apiclient):
     assert result.status_code == status.HTTP_204_NO_CONTENT
 
 
-def test_collection_detail_permission_as_superuser(logged_in_apiclient):
```

Comment:
It might be helpful to keep these tests. I think they should still work if
`url = reverse('collection-detail', kwargs={'collection_key': collection.hexkey})` 
is replaced with 
`url = '/collections/{}'.format(collection.hexkey)`

Reply:
The tests for the collection detail API are testing the same things, and that is what the react view uses to fetch collection detail data. It would be redundant to keep this around (unless I'm missing something)

Reply:
okay